### DISCUSSION TITLE
本地dist路径怎么填写？

### DISCUSSION THREAD

**Discussion by li-peifeng** [AlistGo/alist#4826] [Q&A]
想在本地服务器上托管编译好的dist，可试过多种方式填写cdn地址，都是启动白屏，查看文档也是简单一笔带过，比如我现在是docker版本3.23.0，使用反代端口，dist我放在/www/123/dist，我应该怎么正确填写呢？

**Comment by anwen-anyi** [comment]
直接 默认空着就可以吖,你不是已经编译好了？
```json
  "cdn": "",
```

**Comment by li-peifeng** [comment]
> 直接 默认空着就可以吖,你不是已经编译好了？
> 
> ```json
> 
>   "cdn": "",
> 
> ```

我alist主程序没有自己编译， 是用编译好的dist
